webpack.config.js

```diff
@@ -9,7 +9,10 @@ module.exports = {
       {
         test: /\.tsx?$/,
         use: "ts-loader",
-        exclude: /node_modules/
+        exclude: [
+          /node_modules/,
+          path.resolve(__dirname, "src/__tests__")
```

Comment:
Little but confused by the test data files that were renamed. 
`src/components/DiagramFrame/FramePanels/HelpPanelData/polling_explore.ts` (new name)
Was this just in the wrong place? Want to make sure test data is not in the final build.

Reply:
The removed test data used an older input shape for the `generateModelDiagrams` function that is being tested in diagram_generation. 

Some of that data was also used for the embedded diagrams in HelpPanel.

I updated the help diagram data, and moved it somewhere it could be included in the bundle. 

Now, there is separation between the "example diagrams" in HelpPanelData and the "test diagrams" in `__tests__/`